src/ChipInput.js

```diff
@@ -119,13 +151,22 @@ class ChipInput extends React.Component {
     chips: [],
     focusedChip: null,
     inputValue: ''
-  }
+  };
 
   constructor (props) {
     super(props)
     if (props.defaultValue) {
       this.state.chips = props.defaultValue
     }
+    this.labelRef = React.createRef()
+    this.input = React.createRef()
+  }
+
+  componentDidMount () {
+    if (this.props.variant === 'outlined') {
+      this.labelNode = ReactDOM.findDOMNode(this.labelRef.current)
+      this.forceUpdate()
```

Comment:
Do we really need the raw DOM node and a forced update here? :thinking:

Reply:
I'm not sure. I took this from Material UI's example implementation for the Outlined component. See [here](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/TextField/TextField.js). I'll take it out and see if it still works as expected.

Reply:
:man_shrugging: It's probably fine then. ChipInput always has always been a TextInput, with chips on top.